locale/fr/about/governance.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Gouvernance du Projet
+layout: about.hbs
+---
+# Gouvernance du Projet
+
+## Comité de Pilotage Technique
+
+Le projet est co-dirigé par un Comité de Pilotage Technique
+(Technical Steering Committee - TSC) qui est responsable de
+la gouvernance de haut-niveau du projet.
+
+Le TSC a toute autorité sur ce projet, y compris:
+
+* la Direction Technique
+* La gouvernance et la gestion du projet (y compris cette politique)
+* La politique de contribution
+* La gestion des dépôts GitHub
+* Les guides de Conduite
+* La maintenance de la liste des Collaborateurs
+
+Les invitations originelles à siéger au TSC ont été proposées
+à des contributeurs actifs qui avaient une expérience significative
+en gestion de projet. La participation à ce comité est susceptible
+d'évoluer dans le temps en rapport avec les besoins du projet.
+
+Vous pouvez trouver la liste des membres actuels du TSC sur le projet
+[README.md](https://github.com/nodejs/node/blob/master/README.md#tsc-technical-steering-committee).
+
+## Collaborateurs
+
+Le dépôt GitHub [nodejs/node](https://github.com/nodejs/node) est 
+maintenu par le TSC et un groupe de Collaborateurs additionnels
+qui sont ajoutés par le TSC de manière continue.
+
+Les personnes proposant des contributions significatives sont faits
+Collaborateurs et se voient accorder les droits d'écriture sur le projet.
```

Comment:
> les droits de commit sur

Or, update the `droits de commit` in the next paragraph by `droits en écriture` to keep the same term for both.